src/patternfly/demos/PageTest/examples/page-test-example1.hbs

```diff
@@ -0,0 +1,117 @@
+{{#> page-test}}
+
+{{#> background-image}}
+{{/background-image}}
+
+{{#> page}}
+
+{{#> page-section pf-l-page__main-section--modifiers="pf-m-dark"}}
+Dark content
+{{/page-section}}
+
+{{#> page-section pf-l-page__main-section--modifiers="pf-m-light"}}
+Light content
+{{/page-section}}
+
+{{#> page-section}}
+
+{{#> gallery modifierClass="pf-m-gutter"}}
+  {{#> gallery-item}}
+  {{#> card pf-c-card-modifier="" }}
+    {{#> card-body pf-c-card__content-modifier="" }}
```

Comment:
Do you want the empty modifier on all the cards?

Reply:
not really, I was just copying what was on card component. I'll remove it